packages/TelegramClient-Core.package/TCCCore.class/instance/requestUser.with..st

```diff
@@ -0,0 +1,8 @@
+events
+requestUser: aUserId with: aCallbackID
+
+	self client send: (TCCRequest
+		newWithType: 'getUser'
+		from: {
+		'user_id'->aUserId.
+		'@extra'->aCallbackID})
```

Comment:
The `TCCCore` currently has `getUser:with:` and `requestUser:with:` which have the exact same body. Can we remove one of them?

Reply:
I believe we wanted to rename it and did not delete the old one